src/server.h

```diff
@@ -2231,6 +2236,7 @@ struct redisServer {
     unsigned long long cluster_link_msg_queue_limit_bytes;  /* Memory usage limit on individual link msg queue */
     int cluster_drop_packet_filter; /* Debug config that allows tactically
                                    * dropping packets of a specific type */
+    int cluster_slot_stats_enabled; /* Cluster slot usage statistics tracking enabled. */
```

Comment:
forget to set cluster_slot_stats_enabled to 0 in initServer().

Reply:
Good catch, done.